README.md

```diff
@@ -12,9 +12,9 @@
 -	[Terraform](https://www.terraform.io/downloads.html) 0.12.x
 -	[Go](https://golang.org/doc/install) 1.11 (to build the provider plugin)
 
-## Building The Provider
+## Building the Provider
 
-Clone repository to: `$GOPATH/src/github.com/terraform-providers/terraform-provider-pagerduty`
```

Comment:
This provider path does not exist: github.com/terraform-providers/terraform-provider-pagerduty